app/controllers/greetings_controller.rb

```diff
@@ -0,0 +1,6 @@
+class GreetingsController < ApplicationController
+  def index
+    id = rand(1..Greeting.count)
+    render json: Greeting.find(id)
```

Comment:
React app does not show a random greeting. Kindly make sure that all greetings are randomly rendered when an API GET request is received.

Reply:
Hi @SajjadAhmad14 sorry I don't understand your requirement. I'm following the project requirements `Create an API endpoint that selects a random greeting from your table (you will need a controller with an action and rails route).` selecting a random greeting for each API GET request.